### PR TITLE
Added MikroTik Routers and Switches

### PR DESCRIPTION
**Pull request by baldoarturo**
I added the entire portfolio of MikroTik's routers and switches, updated to 28 Dec 2019
Company products can be seen here: [https://mikrotik.com/products](https://mikrotik.com/products)

Devices powered via a DC adaptor or external power supply don't have power ports as there is no schema for those ports

### CODE REVIEW

File: device-types/MikroTik/CCR1009-7G-1C-1S+.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+console-ports:
+  - name: serial0
+    type: de-9
+power-ports:
+  - name: power1
+    type: iec-60320-c14
+    maximum_draw: 20
+  - name: power2
+    type: iec-60320-c14
+    maximum_draw: 20
```
Comment (jeremystretch) [inline device-types/MikroTik/CCR1009-7G-1C-1S+.yaml]: Please ensure that each file ends with a newline. (Tests are being updated to enforce this.) This ensures clean output when viewing files on a command prompt.

File: device-types/MikroTik/CCR1016-12S-1S+.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    type: 1000base-x-sfp
+  - name: sfp9
+    type: 1000base-x-sfp
+  - name: sfp10
+    type: 1000base-x-sfp
+  - name: sfp11
+    type: 1000base-x-sfp
+  - name: sfp12
+    type: 1000base-x-sfp
+  - name: sfp-sfpplus1
```
Comment (jeremystretch) [inline device-types/MikroTik/CCR1016-12S-1S+.yaml]: Is this the actual interface name? I'm not familiar with Mikrotik so just checking.

Comment (baldoarturo) [reply device-types/MikroTik/CCR1016-12S-1S+.yaml]: Thanks for your fast reply Jeremy. MikroTik names their interfaces like that (yep, I know it's weird). This is an example output from a router I have here
```
[rootmin@ccr1072] > inte eth pr
Flags: X - disabled, R - running, S - slave
 #    NAME                             MTU MAC-ADDRESS       ARP             SWITCH
 0    ether1                          1500 6C:3B:6B:E2:3E:5F enabled
 1 R  sfp-sfpplus1                    1500 6C:3B:6B:E2:3E:57 enabled
 2    sfp-sfpplus2                    1500 6C:3B:6B:E2:3E:58 enabled
 3 R  sfp-sfpplus3                    1500 6C:3B:6B:E2:3E:59 enabled
 4    sfp-sfpplus4                    1500 6C:3B:6B:E2:3E:5A enabled
 5    sfp-sfpplus5                    1500 6C:3B:6B:E2:3E:5B enabled
 6    sfp-sfpplus6                    1500 6C:3B:6B:E2:3E:5C enabled
 7    sfp-sfpplus7                    1500 6C:3B:6B:E2:3E:5D enabled
 8    sfp-sfpplus8                    1500 6C:3B:6B:E2:3E:5E enabled
```

File: device-types/MikroTik/CCR1009-7G-1C-1S+.yaml (hunk 1 above)
Comment (baldoarturo) [reply device-types/MikroTik/CCR1009-7G-1C-1S+.yaml]: Fixed on 4ef0de5

**Pull request merged**